### ISSUE TITLE
Dead sensor bug

### ISSUE THREAD

**Issue by CCCP84** [CnCNet/cncnet-yr-client-package#305]
There is a known bug where a killed sensor unit in the sea leaves behind a certain area in which the player continues to see hidden underwater enemy units. Anyone who has played sea maps has probably encountered this. This is a fairly common occurrence.
Perhaps this can be fixed?

**Comment by CCCP84** [comment]
As a version, it is possible that the unit was killed while moving, and the boundaries of the removed sensor viewing area did not coincide with the actual sensor viewing area at the moment. A solution may be to increase the radius of the removed sensor viewing area. Of course, if possible.

**Comment by Belonit** [comment]
I'm not sure, but maybe Ares fixes this Issues. Have you seen this in the beta version?

**Comment by CCCP84** [comment]
No. We tried to organize a large game for the BETA test, however, one of the participants did not load during all attempts.
Can't confirm this for BETA now.

**Comment by CCCP84** [comment]
[The logic associated with this flag suffers from an easily exploitable bug. When a unit with SensorsSight is destroyed, a permenantly viable sensor area is retained, centered around the point of that unit's destruction.](https://modenc.renegadeprojects.com/SensorsSight)
